src/lib/auth/utils.js

```diff
@@ -10,3 +14,27 @@ export const getIsAuthenticated = idTokenContents =>
     !!idTokenContents && Math.round((new Date()).getTime() / 1000) < idTokenContents.exp;
 
 export const makeAuthorizationHeader = token => token ? {"Authorization": `Bearer ${token}`} : {};
+
+// TODO: move hooks to own file
+
+export const useAuthorizationHeader = () => {
+    const {accessToken} = useSelector(state => state.auth);
+    return useMemo(
+        () => accessToken => accessToken ? {"Authorization": `Bearer ${accessToken}`} : {},
+        [accessToken]);
+};
+
+export const useResourcePermissions = (resource) => {
```

Comment:
Nice addition for authz! 
Minor suggestion (could be a todo comment): It might be worthwhile to extend this hook to a `hasResourcePermissions => (resource, permission)` that evaluates if the permission is valid or not.